### PR TITLE
LPS-146630 Update Selected Quote - Select Payment Method to Mobile and Tablet View

### PR DESCRIPTION
**Pull request by samuelleolopes**

### CODE REVIEW

**Comment by samuelleolopes** [comment]
## :fire: GitRay Report 
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
 ## Jira Tasks 

 <html><details><summary>More details about: <b>LPS-146630 Update Selected Quote - Select Payment Method to Mobile and Tablet View</b>.</summary>
              <br />
              <table>
              <tr>
            <th>Title</th>
            <td><a target="_blank" href="https://issues.liferay.com/browse/LPS-146630">Update Selected Quote - Select Payment Method to Mobile and Tablet View</a></td>
        </tr>
              <tr>
            <th>Type</th>
            <td>Technical Task</td>
        </tr>
              <tr>
            <th>Priority</th>
            <td>Média</td>
        </tr>
              <tr>
            <th>Sub-Tasks</th>
            <td>No Sub-Tasks Opened</td>
        </tr>
              <tr>
            <th>Status</th>
            <td>Em Andamento</td>
        </tr>
              <tr>
            <th>Assignee</th>
            <td>Samuel Lopes</td>
        </tr>
              <tr>
            <th>Reporter</th>
            <td>Samuel Lopes</td>
        </tr>
                </table>
          </details></html> 

---

Hey, don't forget to update all related tickets with correct status, such as **In Review** :wink: 
### Git Stats 

2 files changed, 20 insertions(+), 12 deletions(-)

**Comment by samuelleolopes** [comment]
ci:test:sf

**Comment by samuelleolopes** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/common/styles/apps/selected-quote/components/steps/payment-method.scss [hunk 1]
```diff
@@ -38,3 +38,9 @@
 		padding-bottom: 13px;
 	}
 }
+
+.billing-options {
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/common/styles/apps/selected-quote/components/steps/payment-method.scss:42]: Could you make this class more especific?

Example:
```css
.class-name .billing-options {
    ...
}
```

File: modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/selected-quote/components/Steps/PaymentMethod/index.js [hunk 2]
```diff
@@ -204,12 +204,12 @@ const PaymentMethod = () => {
 
 						{checkedMethod.options.length ? (
 							<>
-								<div className="c-mb-3 d-flex flex-row">
+								<div className="c-mb c-px-3 d-flex row">
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/selected-quote/components/Steps/PaymentMethod/index.js:207]: i think you forgot the ```c-mb``` class here. Was that?

**Review by silasbsi — CHANGES_REQUESTED** [review]
There are some adjustments you need to make

**Comment by samuelleolopes** [comment]
ci:test:sf

**Review by silasbsi — APPROVED** [review]

**Comment by silasbsi** [comment]
Thanks @samuelleolopes . @kevenleone could you make another review?

**Comment by kevenleone** [comment]
LGTM 🔥

**Comment by kevenleone** [comment]
ci:forward

**Review by kevenleone — APPROVED** [review]

**Pull request closed without merging**